tests/posix_mqueue/Makefile

```diff
@@ -0,0 +1,6 @@
+TARGETS=$(patsubst %.c,%,$(wildcard *.c))
+LDLIBS += -lselinux -lrt
+test_sigiotask: LDLIBS += -lutil
```

Comment:
Looks like cut-and-paste for LDLIBS and test_sigiotask lines, unnecessary AFAICT for your test programs.